### ISSUE TITLE
Print dialogue

### ISSUE THREAD

**Issue by gedw99** [ncruces/zenity#58]
I think we can build a Print dialogue for golang apps.

So then you can send a pdf or Image to Print dialogue.

I investigated it a year ago but there was not zenith back then that can do no cgo which will make it much easier .

Thanks @gen2brain and @ncruces for this great project 

@gen2brain built a printing api but it did not bring up the dialogue if I recall

**Comment by gedw99** [comment]
Also I think there is a call back flow from the print dialogue to your app making the pdf . When you change orientation or page size it needs to ask your app for a pdf that is that size. 

Also there is no reason it has to be a pdf but it’s just the lowest common denominator to get going i feel

**Comment by gedw99** [comment]
The api was reasonably easy but I got blocked on thhe cgo cross platform. 

Would like to start off my adding the API notes as a PR

**Comment by ncruces** [comment]
Problems:
- `zenity` does not have a print dialog
- does `osascript`?

These two, I feel, make this a poor fit for this package/module.

**Comment by ncruces** [comment]
Closing this as out-of-scope for the package.

**Comment by gedw99** [comment]
no worries

if you have any suggestions let me know.

have printing working via apple script :)
